apps/vr-tests/src/stories/ScrollablePane.Sticky.DetailsList.stories.tsx

```diff
@@ -0,0 +1,267 @@
+import { getTheme, mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+
+const classNames = mergeStyleSets({
+  wrapper: {
+    height: '80vh',
+    position: 'relative'
+  },
+  filter: {
+    paddingBottom: 20,
+    maxWidth: 300
+  },
+  header: {
+    margin: 0
+  },
+  row: {
+    display: 'inline-block'
+  }
+});
+
+const _footerItem: IScrollablePaneDetailsListExampleItem = {
+  key: 'footer',
+  test1: 'Footer 1',
+  test2: 'Footer 2',
+  test3: 'Footer 3'
+};
+
+export interface IScrollablePaneDetailsListExampleItem {
+  key: number | string;
+  test1: string;
+  test2: string;
+  test3: string;
+}
+
+export interface IScrollablePaneDetailsListExampleState {
+  items: IScrollablePaneDetailsListExampleItem[];
+}
+
+export class ScrollablePaneDetailsListExample extends React.Component<
+  {},
+  IScrollablePaneDetailsListExampleState
+> {
+  private _selection: Selection;
+  private _allItems: IScrollablePaneDetailsListExampleItem[];
+  private _columns: IColumn[];
+
+  constructor(props: {}) {
+    super(props);
+
+    this._selection = new Selection();
+
+    this._allItems = [];
+    let rowData = '';
+    for (let i = 0; i < 200; i++) {
+      rowData = 'row ' + (i + 1).toString() + ', column ';
+      this._allItems.push({
+        key: i,
+        test1: rowData + '1',
+        test2: rowData + '2',
+        test3: rowData + '3'
+      });
+    }
+
+    this._columns = [];
+    for (let i = 1; i < 4; i++) {
+      this._columns.push({
+        key: 'column' + i,
+        name: 'Test ' + i,
+        fieldName: 'test' + i,
+        minWidth: 100,
+        maxWidth: 200,
+        isResizable: true
+      });
+    }
+
+    this.state = {
+      items: this._allItems
+    };
+  }
+
+  public render(): JSX.Element {
+    const { items } = this.state;
+    const stickyAboveContainer: IStickyContainerBehavior = {
+      arrangeStickiesBasedOnOrder: true,
+      containerBehavior: StickyContainerBehaviorType.StickyOnScroll,
+      notUsePlaceHolder: true
+    };
+    const stickyBelowContainer: IStickyContainerBehavior = {
+      arrangeStickiesBasedOnOrder: true,
+      containerBehavior: StickyContainerBehaviorType.StickyAlways,
+      notUsePlaceHolder: true
+    };
+    const stickyBackgroundColor = getTheme().palette.white;
+    return (
+      <div
+        style={{
+          height: '80vh',
+          position: 'relative',
+          maxHeight: 'inherit',
+          width: '900px'
+        }}
+      >
+        <Fabric>
+          <div className={classNames.wrapper}>
+            <ScrollablePane
+              style={{ maxWidth: '900px', border: '1px solid #edebe9' }}
+              scrollbarVisibility={ScrollbarVisibility.always}
+              stickyHeaderContainerBehavior={stickyAboveContainer}
+              stickyFooterContainerBehavior={stickyBelowContainer}
+            >
+              <Sticky
+                stickyPosition={StickyPositionType.Header}
+                order={1}
```

Comment:
Design question: We know the order of elements in JSX on initial render. Can the control not infer based off that without requiring the consumer to supply it explicitly?

Reply:
I don't think it would work if there is a component which is defer loaded or conditionally rendered.